sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -68,6 +68,33 @@ It's important to keep the `TracerProvider`, `MeterProvider`, and `LoggerFactory
 
   For a complete example see [LogDemo.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Logs/LogDemo.cs).
 
+### Add the Exporter (single api for all signals)
```

Comment:
Borrow the idea from here for readme too - https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md#enable-otlp-exporter-for-all-signals

Reply:
I made changes, please re-review.

I removed the example of manually start the Hosted Services, but included a warning. Let me know what you think.
I'm concerned about supportability costs if users can't discover this on their own.

Reply:
> I'm concerned about supportability costs if users can't discover this on their own.

Detailed documentation should be covered in learn article.